docs/getting-started.rst

```diff
@@ -93,6 +93,8 @@ Required columns in input data
 +--------------+---------------------------------------------------------+-------------+
 |pkv           |Whether individual is (only) privately health insured    | Bool        |
 +--------------+---------------------------------------------------------+-------------+
+|priv_pens_exp |Monthly contributions to private pension insurance       | Float       |
```

Comment:
`priv_pens_contr`

Reply:
I went for `pr_pens_contr`...I don't feel like extending the column width

Reply:
Done in 24ca3034 (<1min with Atom + sublime-style-column-selection :-)). Please add the `r` in the code.